composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "bd93faf2a75150f9ba8e41fb86b8c632",
+    "content-hash": "2723c5a59596e6ccf2430f75a1fcf36b",
```

Comment:
What command did you please use to update composer.lock here? I need it so I can replay the changes on 1.11.x

Reply:
`composer update --lock`, see [docs](https://getcomposer.org/doc/03-cli.md#update-u-upgrade)

Reply:
Thanks!

Reply:
Just realized this advice is wrong.

Example:

1) 1.10.x updates `phpstan/phpdoc-parser` from 1.25.0 to 1.26.0.
2) I merge 1.10.x into 1.11.x and get conflict in composer.lock
3) I run `git checkout --ours composer.lock` to get it look like on top of 1.11.x.

Now I need to rerun the commands that make the lock file being changed in 1.10.x. Running `composer update --lock` is not sufficient, it's going to keep phpdoc-parser on version 1.25.0 in composer.lock. But I want it to be 1.26.0.

The correct solution is to run `composer update phpdoc-parser`.

Reply:
My working flow:
- I merge all non-conflicting stuff of `composer.lock` in PHPStorm merge tool.
- The only conflict is the hash, I pick any.
   - _If not, some package got update on both sides._
- I fix that hash by `composer update --lock`

Reply:
I see!